skyportal/handlers/source.py

```diff
@@ -206,10 +206,10 @@ def post(self):
             end_date = arrow.get(data['endDate'].strip())
             q = q.filter(Source.last_detected <= end_date)
         if data['simbadClass']:
-            q = q.filter(func.lower(Source.simbad_class) ==
-                         data['simbadClass'].lower())
+            q = q.filter(func.lower(Source.external_metadata['simbad']['class'].astext)
```

Comment:
Does this work if `simbad` is not in `external_metadata`?

Reply:
@stefanv yes, this works even when `simbad` is not in `external_metadata` for some or all sources.